server/go.mod

```diff
@@ -8,8 +8,10 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/stellar/go v0.0.0-20230316102104-335848c1cd8e
 	github.com/threefoldtech/grid3-go v1.0.1
-	github.com/threefoldtech/substrate-client v0.1.4
-	golang.org/x/net v0.8.0
+	github.com/threefoldtech/substrate-client v0.1.5
+	github.com/threefoldtech/tf-grid-cli v1.1.3-0.20230330203613-f63826e9258e
```

Comment:
why is this depedency added?

Reply:
no sure. changing the dependencies from `grid-proxy-server` to `tfgrid-sdk-go/grid-proxy` updated it. will check it

Reply:
It was added to make the server build. That dependency is still needed from the old development. But we will be merging the tfgrid branch first (the dependency to tf-grid-cli has be removed completely there). After that we'll rebase this branch and merge this one.